cloud_cleaner.sh

```diff
@@ -354,7 +354,8 @@ GCP_PROJECT=$(jq -r '.project_id' "$GOOGLE_APPLICATION_CREDENTIALS")
 $GCP_CMD config set project "$GCP_PROJECT"
 
 # List tagged intances and remove the old enough ones
-INSTANCES=$($GCP_CMD compute instances list --filter='labels.gitlab-ci-test:true' \
+echo -e "------------------\nCleaning instances\n------------------"
+INSTANCES=$($GCP_CMD compute instances list --filter='NOT labels.persist:true AND NOT labels.persist:true' \
```

Comment:
Why is it checking twice for `NOT labels.persis:true` ?

Reply:
oh wow, my intention was to check `labels` and `tags`, let me fix it real quick